framework/include/bcs/ADIntegratedBC.h

```diff
@@ -24,10 +24,10 @@ class ADIntegratedBCTempl : public IntegratedBCBase, public MooseVariableInterfa
 
   virtual MooseVariableFE<T> & variable() override { return _var; }
 
-  void computeResidual() override;
-  void computeJacobian() override;
-  void computeJacobianBlock(MooseVariableFEBase & jvar) override;
-  void computeJacobianBlockScalar(unsigned int jvar) override;
+  virtual void computeResidual() override;
```

Comment:
Looks like this is unnecessary after some googling...what's the moose standard on this?

Reply:
What do you mean by unnecessary? We are implementing `computeResidual` in this class template and it's overriding the method in the base class.

Reply:
I mean that the `virtual` keyword is unnecessary, which is the diff here.